bin/main.rb

```diff
@@ -1,4 +1,20 @@
 require '../lib/game'
 require '../lib/board'
 
-Game.new
+game = Game.new
+
+game_end = false
+until game_end
+
+  input = false
+  until input
+    game.move # show the board and the instructions
+    field = gets.chomp
+    input = game.input(field) # ask for input and sanitize
+  end
+
+  play = game.play(input)
+
+  game_end = game.check_game if play
+
+end
```

Comment:
Insert lines 6 to 20 in a method inside `Game`. Call it `start` maybe? You can then do `game.start`